### DISCUSSION TITLE
Status of this fork: Official continuation of OpenARC, or something else?

### DISCUSSION THREAD

**Discussion by edmundlod** [flowerysong/OpenARC#22] [General]
Hi @flowerysong, I have been wondering if this fork is the official continuation of OpenARC (I saw that you had been contacted by them for a paid job of sorts) {update: Clearly not, my mistake!]? Or is this a testing ground that will get merged into the original OpenARC repo (by trusteddomains)? Or is this just "your" fork, and will it continue with the same name, or perhaps under a new name?

I am maintaining the official OpenARC package on Archlinux AUR, as well as your fork (which is the one that is promoted on the Archlinux wiki). However, the current naming is a bit off, so I was wondering what the status of this fork and of the original repo package is. I hope you can enlighten me / us.

In any event, very many thanks for the ton of work you put into this - it is really great to have this being updated and built out so nicely!

**Comment by flowerysong** [comment]
I have no relationship with the Trusted Domain Project other than minor contributions to their projects in the past, nor have they contacted me regarding this fork or OpenARC in general.

The TDP OpenARC project has been completely dead for half a decade (and not hugely responsive for a year or so before that.) This was originally just an internal fork (with far fewer changes) maintained for my employer because the official release of `libopenarc` was completely unsuitable for production use due to lack of interoperability with Microsoft. There was a burst of community activity on the upstream repository this year that continued to be ignored by the TDP, so I decided to try to keep that momentum going by making our fork public, integrating more of the open PRs, and dedicating a bit of time to addressing the years of neglect.

I cannot predict the future. For now I'm just maintaining this fork, accepting community contributions, and monitoring the TDP repository for new issues and PRs.

**Comment by edmundlod** [comment]
Thank you for your reply, @flowerysong. I apologise for my mistake, I must have misread something! Thank you for keeping this up to date for now :)
